catalyst/README.md

```diff
@@ -153,158 +158,413 @@ whenItem(store.buttonParent)
+    //following line is equivilant
+    //get dom(){ return $$('devicesSection').find('[id*=mvo_ovr_devices]').first().parent();}
+  },
 });
 ```
 
+### rule.whenDom(<selector>)
 
-### rule.whenDOM(<selector>)
+```js
+rule.whenDom('.evolv-DeviceSection');
+```
 
+The "whenDom" method will wait for the specified selector to be created or selectable on the page and will return a Promise object. The Promise object can then be used to apply further methods.
 
 ### rule.whenItem(<instrumention key>)
-    
-    
-rule.whenDom('.evolv-DeviceSection')
-
 
+```js
+rule.whenItem('DeviceSection');
+```
 
+The "whenItem" method will wait for the specified instrument key to be created or selectable on the page and will return a Promise object. The Promise object can then be used to apply further methods.
 
 ### rule.$
-The methods below are mostly analagous to their JavaScript or jQuery counterparts.
 
+The methods below are mostly analagous to their JavaScript or jQuery counterparts. However, this will return a Catalyst wrapped specified selector object.
 
 #### filter()
+
 The `filter()` method creates a new array with all elements that pass the test implemented by the provided function.
-```
+
+```js
 var words = ['spray', 'limit', 'elite', 'exuberant', 'destruction', 'present'];
 var result = $(words).filter((idx, word) => word.length > 6);
 console.log(result);
 // expected output: Array ["exuberant", "destruction", "present"]
 ```
 
 #### find()
+
 The `find()` method returns the all elements that match the specified child selector.
-```
+
+```html
 <ul id="sidebar">
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
 </ul>
+```
+
+```js
 $("#sidebar").find("li");
 // expected output: array of 5 <li> elements.
 ```
 
 #### closest()
+
 The `closest()` method returns the parent element that matches the specified selector.
-```
+
+```html
 <ul id="sidebar">
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
 </ul>
+```
+
+```js
 $("li").closest("#sidebar");
 // expected output: array with single <ul> element.
 ```
 
 #### parent()
+
 The `parent()` method returns the parent element of the specified selector.
-```
+
+```html
 <ul id="sidebar">
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
-<li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
+  <li>Lorum ipsum</li>
 </ul>
+```
+
+```js
 $("li").parent();
 // expected output: array with single <ul> element.
 ```
 
 #### children()
 The `children()` method returns the child elements of the specified selector.
-```
+
+```html
 <ul id="sidebar">
```

Comment:
Nitpick: All of these sidebar examples have an extra space after `id="sidebar"`

watch() example has an extra space after `$('#sidebar').watch(`